programs/cleos/httpc.hpp

```diff
@@ -107,7 +107,9 @@ namespace eosio { namespace client { namespace http {
 
 
    const string history_func_base = "/v1/history";
+   const string trace_api_func_base = "/v1/trace_api";
```

Comment:
Update `status_code == 404` section at end of `httpc.cpp` to indicate that Trace API plugin is not enabled.